website/static/js/passwordForms.js

```diff
@@ -92,6 +92,31 @@ var BaseViewModel = oop.extend(ChangeMessageMixin, {
 var ChangePasswordViewModel = oop.extend(BaseViewModel, {
     constructor: function () {
         var self = this;
+
+        // Call constructor at the beginning so that self.password exists
+        self.super.constructor.call(this);
```

Comment:
Since ChangePasswordViewModel does override getValidatedObservables, you will need to make sure that `self.validatedFields`  gets populated correctly.